privaterelay/templates/includes/remaining-aliases.html

```diff
@@ -1,17 +1,24 @@
 {% load static %}
 {% load relay_tags %}
+{% load ftl %}
+{% ftlconf bundle='privaterelay.ftl_bundles.main' %}
 
+{% remaining_free_aliases relay_addresses as remaining_free_aliases_count %}
```

Comment:
This variable is set to capture the `simple_tag` function call and the output is passed as an argument to the translation string.